Duplicati/Library/Main/Database/Database schema/12. Performance Indexes.sql

```diff
@@ -0,0 +1,12 @@
+CREATE INDEX "nnc_Metadataset" ON Metadataset ("ID","BlocksetID");
+CREATE INDEX "nn_FilesetentryFile" on FilesetEntry ("FilesetID","FileID");
```

Comment:
I don't think we have a convention for the index names, but is there a meaning behind the `nnc_` and `nn_` prefixes?

![image](https://user-images.githubusercontent.com/17345532/156477520-0b52f32c-1d01-4a0e-8422-9f57b6052c36.png)

Reply:
Ya, nn = Non Unique non clustered, nc would be non unique clustered, the nnc stands for non-unique non-clustered and covered (covered meaning it has all the columns in a table in the index).

Reply:
Sounds good, thanks!